bin/harbor-restore.sh

```diff
@@ -165,7 +165,11 @@ log_info "Cleaning up restore artifacts..."
 "${root_path}/bin/ck8s" ops kubectl sc delete -n harbor -f "${TMP_JOB_FILE}"
 "${root_path}/bin/ck8s" ops kubectl sc delete configmap -n harbor restore-harbor
 
+# Reinitialize
+"${root_path}/bin/ck8s" ops kubectl sc delete job -n harbor init-harbor-job
+"${root_path}/bin/ck8s" ops helmfile sc sync -l app=harbor
+"${root_path}/bin/ck8s" ops kubectl sc wait --for=condition=complete job -n harbor init-harbor-job --timeout=-1s
```

Comment:
I believe that if the job's pod fails this will get stuck forever. Give it a decent amount of timeout instead? Perhaps with a "please check the logs using this command..." if it fails?

Reply:
84885604